nxviz/plots.py

```diff
@@ -338,10 +340,18 @@ def __init__(self, graph, **kwargs):
         Accepts the following additional arguments apart from the ones in
         `BasePlot`:
 
-        :param rotate_labels: Whether to rotate node labels.
-        :type node_color: `bool`
+        :param node_layout: which/whether (a) node layout is used,
+            either 'rotation', 'numbers' or None
+        :type node_layout: `string`
         """
-        self.rotate = kwargs.pop("rotate_labels", False)
+
+        # Store node layout
+        specified_layout = kwargs.pop("node_layout", None)
+        valid_node_layouts = (None, 'rotation', 'numbers')
+        assert specified_layout in valid_node_layouts
```

Comment:
Please change `valid_node_layouts` to `valid_node_label_layouts`. I will leave it to you to decide whether or not to change `specified_layout` to `specified_node_layout` or not.